.github/workflows/build.yaml

```diff
@@ -57,4 +57,5 @@ jobs:
         tools/ci/run_ci.sh \
           --run-clean \
           --run-build \
-          --source-directory examples/${{ matrix.example }}
+          --source-directory examples/${{ matrix.example }} \
+          --build-directory wakaama-build-${{ matrix.example }}
```

Comment:
I can not comment on the commit message itself, so I do it here instead: What is the goal of this change? Will it allow e.g. for multiple builds ending up in the same CI artifact? Or is this just an improvement for local development?

Reply:
This way you can build the examples separately without overwriting previous builds. It's useful for local development. But it will mainly be used when we refactor the examples to be individual projects, that depend on Wakaama as a 3rd-party.
Currently, the examples are tightly coupled to the rest of the project.

Reply:
I think this comment would be good to have in the commit message.

Reply:
You are right! I'll add some clarification to the commit.